docs/docs/advanced/air-gap.md

```diff
@@ -1,142 +1,125 @@
+## Network requirements
+Trivy's Databases are distributed as OCI images via GitHub Container registry (GHCR):
 
-## Air-Gapped Environment for vulnerabilities
+- <https://ghcr.io/aquasecurity/trivy-db>
+- <https://ghcr.io/aquasecurity/trivy-java-db>
+- <https://ghcr.io/aquasecurity/trivy-checks>
 
-### Download the vulnerability database
-At first, you need to download the vulnerability database for use in air-gapped environments.
+If Trivy is running behind a firewall, you'll need to add the following urls to your allowlist:
 
-=== "Trivy"
+- `ghcr.io`
+- `pkg-containers.githubusercontent.com`
 
-    ```
-    TRIVY_TEMP_DIR=$(mktemp -d)
-    trivy --cache-dir $TRIVY_TEMP_DIR image --download-db-only
-    tar -cf ./db.tar.gz -C $TRIVY_TEMP_DIR/db metadata.json trivy.db
-    rm -rf $TRIVY_TEMP_DIR
-    ```
+The databases are pulled by Trivy using the [OCI Distribution](https://github.com/opencontainers/distribution-spec) specification, which is based on simple HTTPS protocol.
 
-=== "oras >= v0.13.0"
-    Please follow [oras installation instruction][oras].
+## Running Trivy in air-gapped environment
+In an air-gapped environment, you have to tell Trivy on every scan to not attempt to download the latest database files, otherwise the scan will fail. The following flags are relevant:
 
-    Download `db.tar.gz`:
+- `--skip-db-update` to skip updating the main vulnerability database.
+- `--skip-java-db-update` to skip updating the Java vulnerability database.
+- `--offline-scan` to scan Java applications without issuing API requests.
+- `--skip-check-update` to skip updating the misconfiguration database.
 
-    ```
-    $ oras pull ghcr.io/aquasecurity/trivy-db:2
-    ```
-
-=== "oras < v0.13.0"
-    Please follow [oras installation instruction][oras].
+```shell
+trivy image --skip-db-update --skip-java-db-update --offline-scan --skip-check-update myimage
+```
 
-    Download `db.tar.gz`:
+## Self-Hosting
+You can also host the databases on your own OCI registry, in order to avoid having Trivy reaching out of your network.  
 
-    ```
-    $ oras pull -a ghcr.io/aquasecurity/trivy-db:2
-    ```
+First, make a copy of the databases in a container registry that is accessible to Trivy. The databases are in:
+- `ghcr.io/aquasecurity/trivy-db:2`
+- `ghcr.io/aquasecurity/trivy-java-db:1`
+-  `ghcr.io/aquasecurity/trivy-checks:0`
 
-### Download the Java index database[^1]
-Java users also need to download the Java index database for use in air-gapped environments.
+Then, tell Trivy to use the private images:
 
-!!! note
-    You container image may contain JAR files even though you don't use Java directly.
-    In that case, you also need to download the Java index database.
+```shell
+trivy image \
+    --db-repository myregistry.local/trivy-db \
+    --java-db-repository myregistry.local/trivy-java-db \
+    --offline-scan \
+    --checks-bundle-repository myregistry.local/trivy-checks \
+    myimage
+```
 
-=== "Trivy"
+### Authentication
```

Comment:
I agree, I also thought about the allowlist and private access not belonging to the airgap document, but also didn't want to leave them in a vulnerability specific document since they are shared to misconfig and maybe other things in the future.
WDYT about renaming the air-gap document to "advanced database scenarios"? then it makes more sense why the network requirements, authentication, and offline settings are there. 
It does sounds similar to the existing "configuration/db.md" doc that you linked, but I thought that the entire "configuration" section is purely CLI flags reference (and I thought it's auto-generated)

Reply:
>WDYT about renaming the air-gap document to "advanced database scenarios"? then it makes more sense why the network requirements, authentication, and offline settings are there.

Yes, It makes more sense, but since we also have checks bundle, advanced "database" scenarios might be misleading.

Reply:
https://github.com/aquasecurity/trivy/pull/7160/commits/4695eebf7a876ff18c9a56f07e4ef4c141dc7e10